### PR TITLE
Updated terraform plugin testing to v1.10.0

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,25 +5,25 @@ go 1.21
 toolchain go1.21.5
 
 require (
-	github.com/ciscoecosystem/aci-go-client/v2 v2.30.0
+	github.com/ciscoecosystem/aci-go-client/v2 v2.31.0
```
Comment (samiib) [inline go.mod:8]: Maybe the commit message should also mention this update?

Comment (gmicol) [reply go.mod:8]: I do agree with @samiib. We should add this change in the commit message.

Comment (shrsr) [reply go.mod:8]: Done!

**Review by gmicol — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by samiib — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**